### PR TITLE
dashboards/cypress-lighthouse

### PR DESCRIPTION
**Pull request by goransle**
Adds comparison of  between master and PR branch. See the [run summary](https://github.com/highcharts/highcharts/actions/runs/5950509225?pr=19590) for example output.

### CODE REVIEW

File: test/cypress/performance/minimal.cy.js [hunk 1]
```diff
@@ -18,11 +18,11 @@ const desktopConfig = {
...
 
 describe('Lighthouse tests', () => {
-    it('Climate demo', () => {
-        cy.visit('/dashboards/demo/climate');
+    it('Minimal demo', () => {
+        cy.visit('/dashboards/demo/minimal');
         cy.lighthouse(
             {
-                performance: 42
+                performance: 0
```
Comment (karolkolodziej) [inline test/cypress/performance/minimal.cy.js:25]: Is this correct?

Comment (goransle) [reply test/cypress/performance/minimal.cy.js:25]: Kind of, seems you have to set something other than `undefined` there to only run the performance tests. (Could not provide the whole object, but then other categories are also run).

The test will fail [if this comparison](https://github.com/highcharts/highcharts/pull/19590/files#diff-cf22095590be315f44cba8f347076fdf3036fb674ed60d85422d762f7085d9c0R50) is not met, so this threshold is not really needed.

**Review by sebastianbochan — APPROVED** [review]

**Review by pawellysy — APPROVED** [review]

**Comment by pawellysy** [comment]
Looks good! Just one comment. Shouldn't the results be posted on the PR?

**Comment by goransle** [comment]
> Looks good! Just one comment. Shouldn't the results be posted on the PR?

The results are currently only in the actions summary, as we don't have tokens for commenting in the GH secrets quite yet.

**Pull request merged**